spec/models/category_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Category, type: :model do
+  subject { described_class.new }
+
+  describe "Validations" do
+    it "is valid with valid name" do
```

Comment:
Use single quotes when you don't need string interpolation or you don't have a quote inside the string. Also, integrate RuboCop so it will tell you about those things :)